public/script.js

```diff
@@ -6,6 +6,38 @@ $( document ).ready(function() {
   canvas.height = window.innerHeight;
   drawStuff();
 
+  $("#download").click(function() {
+    var outputCanvas = document.createElement("canvas");
+    outputCanvas.width = canvas.width;
+    outputCanvas.height = canvas.height;
+
+    var outputContext = outputCanvas.getContext("2d");
+    outputContext.font = "bold 24pt Times";
+
+    var gradient = outputContext.createLinearGradient(0, 0, canvas.width, 0);
+    gradient.addColorStop(0, "#ECF9FF");
```

Comment:
seems to work fine though
![advice-2](https://cloud.githubusercontent.com/assets/1615322/8848703/930614fc-318d-11e5-855e-281eb5a2acd1.png)